pytential/qbx/target_specific/impl.pyx

```diff
@@ -129,7 +129,7 @@ def h3dall_wrapper(nterms, z, scale, hs, hders):
 
 # {{{ helpers
 
-cdef void legvals(double x, int n, double[] vals, double[] derivs) nogil:
+cdef void legvals(double x, int n, double[] vals, double[] derivs) noexcept nogil:
```

Comment:
No idea if adding `noexcept` to these is reasonable. From a quick skim, it doesn't seem like anything in there would raise an exception.. :\

Reply:
I think that's fine. I'd like to remove that code before too long anyway, so if it still passes tests, that's good enough for me.